hummingbot/connector/exchange/ndax/ndax_constants.py

```diff
@@ -31,8 +34,132 @@
 SUBSCRIBE_ACCOUNT_EVENTS_ENDPOINT_NAME = "SubscribeAccountEvents"
 WS_ORDER_BOOK_CHANNEL = "SubscribeLevel2"
 WS_PING_REQUEST = "Ping"
+WS_PING_ID = "WSPing"
 
 # WebSocket Message Events
 WS_ORDER_BOOK_L2_UPDATE_EVENT = "Level2UpdateEvent"
 
 API_LIMIT_REACHED_ERROR_MESSAGE = "TOO MANY REQUESTS"
+
+MINUTE = 60
+HTTP_ENDPOINTS_LIMIT = "AllHTTP"
+HTTP_LIMIT = 600
+WS_AUTH_LIMIT = "AllWsAuth"
+WS_ENDPOINTS_LIMIT = "AllWs"
```

Comment:
Would prefer these constants to end with `LIMIT_ID`.

Reply:
Yes, good point. Changed